src/unitxt/type_utils.py

```diff
@@ -7,6 +7,114 @@
 from .utils import safe_eval
 
 
+def convert_union_type(type_string: str) -> str:
```

Comment:
Hi @pawelknes , is the | allowed from python 3.9 or from 3.10?
I see this: https://peps.python.org/pep-0604/